src/um/d3d12.rs

```diff
@@ -622,64 +640,73 @@ STRUCT!{struct D3D12_RASTERIZER_DESC {
     ForcedSampleCount: UINT,
     ConservativeRaster: D3D12_CONSERVATIVE_RASTERIZATION_MODE,
 }}
+
 RIDL!{#[uuid(0xc4fec28f, 0x7966, 0x4e95, 0x9f, 0x94, 0xf4, 0x31, 0xcb, 0x56, 0xc3, 0xb8)]
 interface ID3D12Object(ID3D12ObjectVtbl): IUnknown(IUnknownVtbl) {
     fn GetPrivateData(
-        guid: REFGUID,
```

Comment:
Despite `REFGUID` being a macro and not a proper typedef in the original headers, I treat it as a proper typedef in winapi.